README.md

```diff
@@ -42,7 +42,7 @@ You then need to load the plugin. In `boostrap.php`, something like:
 For a complete setup, add the following to your `App\Controller\AppController`:
 
 ```php
-public $components = ['BootstrapUI.Flash'];
+public $theme = 'BootstrapUI';
```

Comment:
It definitely could be a theme type of plugin but then we really need to have more pre-themed stuff ready. 

As I said, thus far it only has some helpers; but it doesn't for example have bake templates for grids, views, forms, etc. By taking the 'theme' spot in an app, I believe the plugin then **has** to offer those at least, as in *not optional*, or it would suck as a 'theme'. What do you think?